service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/scheduling/EhrExtractTimeoutScheduler.java

```diff
@@ -42,8 +41,8 @@ public void processEhrExtractAckTimeouts() {
             try {
                 logger().info("Scheduler has started processing EhrExtract list with Ack timeouts");
                 ehrExtractStatusService.updateEhrExtractStatusWithEhrReceivedAckError(ehrExtractStatus.getConversationId(),
-                                                                                      ERROR_CODE,
-                                                                                      ERROR_MESSAGE);
+                                                                                      ACK_TIMEOUT_ERROR.getCode(),
+                                                                                      ACK_TIMEOUT_ERROR.getMessage());
```

Comment:
If you take up my suggestion regarding naming the enumeration to `ErrorCode` I would say line 26 is not required anymore and suggest accessing these like:

`ErrorCode.ACK_TIMEOUT.getCode()` etc.